src/Resources/views/includes/sidebar/_profile.html.twig

```diff
@@ -36,14 +36,14 @@
     >
         <div class="flex flex-col gap-y-2 m-4 -mb-1 px-4 pt-2 pb-3 rounded-sm shadow-md z-10 bg-white ring-1 ring-black ring-opacity-5 focus:outline-none">
             <div class="border-b">
-                <span class="inline-block whatwedo-utility-paragraph text-sm leading-none mb-2">Angemeldet als</span>
-                <div class="flex whatwedo-utility-bold text-neutral-900 mb-2">
+                <span class="inline-block whatwedo-utility-paragraph text-sm leading-none mb-3">Angemeldet als</span>
+                <div class="flex gap-x-2 items-center whatwedo-utility-bold text-neutral-900 mb-3">
                     {% if data.picture_src|default() %}
-                        <img class="inline-block h-8 w-8 md:h-10 md:w-10 rounded-full"
+                        <img class="inline-block h-6 w-6 md:h-8 md:w-8 rounded-full"
```

Comment:
Nit (because not part of the changeset): Is missing an `alt` description. From my point of view, this image is functional, not decorative, therefore should have a description.

Example:

```
alt="Profile picture of {{ data.name_detail ?? data.name }}"
```

Reply:
@renestalder For me the profile picture is purely visual and does not need to be announced since the username is followed right afterwards. That was my take and why I put an empty `alt` attribute there.

Reply:
@marcwieland95 "Visual" doesn't mean "decorative" (just to make that difference clear). Decorative means, this is just to make things nice and shiny. On the other hand, this profile picture is not just to make things nice and shiny. It has a functional value, showing my profile picture. Therefore, interpreting WCAG, I would consider this not be purely decorative.